src/scicloj/02_numpy.clj

```diff
@@ -15,6 +14,14 @@
   (notespace.api/unlisten)
   nil)
 
+^k/hidden
+(defn ravel [t]
```

Comment:
I have no idea what ravel is. I would appreciate a short docstring.

Reply:
Since it wasn't asked for in Chapter 2, I left `ravel` as an implementation detail.  But `ravel` is equivalent to `numpy.ravel` -- it means to provide a linear iteration for an N-dimensional vector.  For instance:

```python3
>>> import numpy as np
>>> np.array(range(9)).reshape([3, 3]).ravel()
array([0, 1, 2, 3, 4, 5, 6, 7, 8])
```

Reply:
ravel has been made more efficient (now just a `->reader`)